src/member.js

```diff
@@ -2,11 +2,11 @@ const BN = require("bn.js")
 const {utils: { isAddress }} = require("web3")
 
 module.exports = class MonoplasmaMember {
-    constructor(name, address, earnings) {
+    constructor(name, address, earnings, active) {
         this.name = name || ""
         this.address = MonoplasmaMember.validateAddress(address)
         this.earnings = earnings ? new BN(earnings) : new BN(0)
-        this.active = true
+        this.active = active || true
```

Comment:
@isentropy this makes it impossible to initialise active to `false`, because naturally `false` is falsey thus the `|| true` will kick in when `active` is `false`. This is *exactly* the gotcha I was referring to in the slack discussion regarding this syntax earlier today. Use `active = false` in the constructor.

Reply:
ok, I'll change. I wonder why the fix works tho. Let me check.

Reply:
maybe there should be a test that requires the active to be `false`... like, just create and check:
```
const m = new MonoplasmaMember("x", "0x1234567890123456789012345678901234567890", 0, false)
assert.strictEqual(m.active, false)
// or later if we change to expect: expect(m.active).toEqual(false)
```

Reply:
Added in 2026855.